.gitignore

```diff
@@ -37,6 +37,11 @@ local.properties
 android/app/src/main/java/com/expensify/chat/generated/
 .cxx/
 
+# VIM
```

Comment:
We skip them in web-e here: https://github.com/Expensify/Web-Expensify/blob/01183e201e730709577798057fa413f189dd5399/.gitignore#L19-L22